cla_backend/settings/base.py

```diff
@@ -52,6 +51,15 @@ def env_var_truthy_intention(name):
 EMAIL_FROM_ADDRESS = "no-reply@civillegaladvice.service.gov.uk"
 DEFAULT_EMAIL_TO = "cla-alerts@digital.justice.gov.uk"
 
+# Want to alter start and end times via environment variables.
+# In case these are not set, default values are set here
+DEFAULT_NON_ROTA_START_TIME_HR = 8
+DEFAULT_NON_ROTA_END_TIME_HR = 17
```

Comment:
DEFAULT_NON_ROTA_END_TIME_HR and DEFAULT_ED_END_TIME_HR are the same values. Do we need two seprate variables? 

Same could be said for DEFAULT_DISCRIM_START_TIME_HR and DEFAULT_NON_ROTA_START_TIME_HR. 

Just feels like we're repeating ourselves.

Reply:
They are currently set to the same value but they could be different as they are used for different categories of provider. Hence we need the option of different variables to allow overrides for the various categories.